docs/conf.py

```diff
@@ -38,7 +38,6 @@
     "sphinx.ext.napoleon",
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
-    "recommonmark",
```

Comment:
Hey @crwilcox, we made this change in https://github.com/googleapis/python-storage/pull/103 due to failures and related documentation:

https://github.com/readthedocs/recommonmark

```
# for Sphinx-1.4 or newer
extensions = ['recommonmark']

# for Sphinx-1.3
from recommonmark.parser import CommonMarkParser

source_parsers = {
    '.md': CommonMarkParser,
}

source_suffix = ['.rst', '.md']
```

I think I misunderstood how to update this file correctly. Could you PTAL when you have a moment?